test/system/user_test.rb

```diff
@@ -0,0 +1,103 @@
+require "application_system_test_case"
+
+class UsersTest < ApplicationSystemTestCase
+  test "Invite, register, confirm, promote, demote, delete" do
+    admin = users(:admin_user)
+    viewer_email = 'viewer@example.com'
+
+    Capybara.using_session(:admin) do
+      visit root_path
+
+      click_on "Login"
+
+      fill_in 'Email', with: admin.email
+      fill_in 'Password', with: 'testpassword'
+      click_on 'Log in'
+
+      assert page.has_content? "Logged in as #{admin.email}"
```

Comment:
Would you please add a message to these assertions, so it’s a little clearer what’s happening if they fail? (Same with the other ones below.)

Reply:
I've added assertion messages to all `assert` and `refute` calls.